src/core/server/opensearch/version_check/ensure_opensearch_version.ts

```diff
@@ -76,6 +76,7 @@ export const getNodeId = async (
      */
     const state = (await internalClient.cluster.state({
       metric: 'nodes',
+      local: true,
```

Comment:
Looking at the comment on this function call, it seems calling cluster manager node is by intention. Shall we adjust the comment if using `local: true` is what we want?

Reply:
Ack, will update the code comment. Thanks for pointing it out.